### DISCUSSION TITLE
Any plans to port the *.sksl Skia c++ object language for Delphi?

### DISCUSSION THREAD

**Discussion by War3Evo** [skia4delphi/skia4delphi#219] [Q&A] [answered]
In the game code for the brick game it uses c++ object code for the game logic.  Is there any plans to port that language to Delphi?

https://skia.org/docs/user/sksl/

**Comment by viniciusfbb** [comment, marked as answer]
the logic and rendering of bricks game is not in c++, it is in the shader language, to run on the GPU, that's why it is so performant. Maybe the logic can be passed to Delphi (CPU), but I don't know if the performance would be the same.
